apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx

```diff
@@ -46,74 +52,37 @@ export default async function Checkout({ params }: { params: CheckoutParams }) {
             )}
           </h2>
 
-          <form
-            action={async () => {
-              'use server';
-              await signIn('fxa');
-            }}
-          >
-            <p className="text-grey-400 text-sm mt-2 pb-4 ">
-              {l10n.getFragmentWithSource(
-                'next-new-user-sign-in-link-2',
-                {
-                  elems: {
-                    a: (
-                      <button className="underline hover:text-grey-400">
-                        Sign in
-                      </button>
-                    ),
-                  },
-                },
-                <>
-                  Already have a Mozilla account?&nbsp;
-                  <button className="underline hover:text-grey-400">
-                    Sign in
-                  </button>
-                </>
-              )}
-            </p>
-          </form>
+          <SignInForm
+            newsletterLabel={cms.commonContent.newsletterLabelTextCode}
+          />
 
-          <div className="p-6 text-center">
-            {/**
-              Temporary Content. This will be replaced in M3b by the Passwordless
-              email signup form.
-            */}
-            <p className="mb-6">{`Current cart email: ${cart.email}`}</p>
-            <form
-              action={async (formData: FormData) => {
-                'use server';
-                const email =
-                  formData.get('email')?.toString() || 'test@example.com';
-                await getApp().getActionsService().updateCart({
-                  cartId: cart.id,
-                  version: cart.version,
-                  cartDetails: {
-                    email,
-                  },
-                });
-                revalidatePath('/');
-              }}
-            >
-              <input
-                type="email"
-                name="email"
-                className="w-full border rounded-md border-black/30 p-3 placeholder:text-grey-500 placeholder:font-normal focus:border focus:!border-black/30 focus:!shadow-[0_0_0_3px_rgba(10,132,255,0.3)] focus-visible:outline-none data-[invalid=true]:border-alert-red data-[invalid=true]:text-alert-red data-[invalid=true]:shadow-inputError"
-              />
-              <BaseButton
-                className="mt-10 w-full"
-                type="submit"
-                variant={ButtonVariant.Primary}
-              >
-                {' '}
-                Set email
-              </BaseButton>
-            </form>
+          <h3 className="font-semibold text-grey-600 text-start">
+            {l10n.getString(
+              'checkout-create-account',
+              'Create a Mozilla account'
+            )}
+          </h3>
+
+          <div className="flex flex-col gap-4 mt-6 mb-10 desktop:flex-row desktop:items-center desktop:justify-center">
+            <button className="w-full flex items-center justify-center bg-[#F9F9FA] border border-grey-300 font-medium rounded text-sm">
```

Comment:
Could this be updated to use the `BaseButton` component instead with a new variant?

These buttons also seem to be a different size than the Figma file. Could it be changed to conform to what's in Figma?

Reply:
I'll make an update to use the `BaseButton` component. As discussed, I'll change them to conform to how these buttons appear in FxA.